src/Controls/src/Core/HybridWebView/HybridWebView.cs

```diff
@@ -15,40 +25,215 @@ public class HybridWebView : View, IHybridWebView
 			BindableProperty.Create(nameof(HybridRoot), typeof(string), typeof(HybridWebView), defaultValue: "wwwroot");
 
 
-		/// <summary>
-		/// Specifies the file within the <see cref="HybridRoot"/> that should be served as the default file. The
-		/// default value is <c>index.html</c>.
-		/// </summary>
+		/// <inheritdoc/>
```

Comment:
What is this inheriting docs from?

Reply:
From the interface member's docs.

Reply:
BTW just to show it works, if I hover in VS over the property with `<inheritdoc />` it correctly shows the docs:

![image](https://github.com/user-attachments/assets/5e1e7234-32b3-44f4-95c0-594a0bfbddff)